libs/fundamental-redirects/index.js

```diff
@@ -1,3 +1,5 @@
+const { VALID_LOCALES, LOCALE_ALIASES } = require("../constants");
```

Comment:
```suggestion
const { DEFAULT_LOCALE, VALID_LOCALES } = require("@yari-internal/constants");
```

Reply:
If I do this, I can't import do this in my test scripts:
```
const { resolveFundamental } = require("./libs/fundamental-redirects");
```

Reply:
It feels more proper to change it, but I guess it really doesn't matter, because even if we moved the constants, the tests would fail until we changed this. Although, perhaps one could argue that requiring it like `../constants` depends on the implementation details of `node_modules`?

Reply:
I have many times typed into a shell or a one-off test script, something like
```
const { VALID_LOCALES, LOCALE_ALIASES } = require("./libs/fundamental-redirects");
```
and I do this from the repo root. That would cease to work if the two of them depend on each not with `@yari-internal` rather than `..`. 
Not a strong argument I admit. But it definitely works. 

Perhaps the biggest bang for buck is to write a Expressjs wrapper that pretends to be Lambda@Edge on localhost.